config/sargans_color_over_srtm1.yaml

```diff
@@ -0,0 +1,8 @@
+map_publisher:
+    ros__parameters:
+        gdal_dataset_path: /vsizip/vsicurl/https://terrain.ardupilot.org/SRTM1/N09E047.hgt.zip/N09E047.hgt
+        gdal_dataset_color_path: "resources/sargans_color.tif"
+        max_map_width: 3601
+        max_map_height: 3601
+        map_origin_latitude: 47.033333
```

Comment:
Why would we want to declare this not from the file but externally? Is this maybe because the map alignment was not correct before https://github.com/ethz-asl/grid_map_geo/pull/36 ?

Reply:
I needed to declare it somewhere and wanted to provide a reasonable default. Another approach would be to have them as launch arguments. Once we add capability to dynamically load based on UAV location, then this can be removed.